src/dmclock_server.h

```diff
@@ -69,7 +69,7 @@ namespace crimson {
     constexpr auto standard_erase_age = std::chrono::seconds(600);
     constexpr auto standard_check_time = std::chrono::seconds(60);
     constexpr auto aggressive_check_time = std::chrono::seconds(5);
-    constexpr uint standard_erase_max = 100;
+    constexpr uint standard_erase_max = 2000;
```

Comment:
Is the idea that you were able to increase this amount due to the optimizations?